content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/frag-helm-infra-mode.md

```diff
@@ -0,0 +1,60 @@
+---
+title: frag-helm-infra-mode
+---
+
+Before you start, make sure:
+
+1. Each account has a **kubeconfig** file that grants access to the deployment target cluster. If you use Amazon EKS, you can run the following command:
+
+   {{< prism lang="bash" >}}
+   aws eks update-kubeconfig --name <target-cluster>
+   {{< /prism >}}
+
+1. Each account has a **kubeconfig** file and a secret created from that file. For example:
+
+   {{< prism lang="bash" >}}
+   kubectl create secret generic kubeconfig --from-file=<path>/.kube/config -n <namespace>
+   {{< /prism >}}
+
+   See the `kubectl create secret generic` [docs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#-em-secret-generic-em-) for command details.
+
+
+Run one of the following commands:
+
+1. If you want to connect to Armory Cloud services:
+
+   You must set your Armory Cloud **clientId** and **clientSecret** credentials.
+
+   You must set your **kubeconfig** file and secret.
+
+   {{< prism lang="bash" line="4" >}}
+   helm install armory-agent armory-charts/agent-k8s-full \
+   --create-namespace \
+   --namespace=<agent-namespace> \
+   --set hub.auth.armory.clientId=<your-clientID> \
+   ,hub.auth.armory.secret=<your-clientSecret> \
+   ,kubeconfigs.<account-name>.file=<kubeconfig> \
```

Comment:
let's specify somewhere that this is the name of the file used when creating the secret.  E.g. if the secret was created with `--from-file=<path>/.kube/config` this value should be **config**